go-controller/pkg/cni/ipv4_hack.go

```diff
@@ -0,0 +1,69 @@
+package cni
+
+// cnishim hack to add an IPv4 interface to certain pods in single-stack IPv6 clusters
```

Comment:
maybe expand this a bit to explain why we need this?

"for AWS API access, which are reachable via IPv4 only"